### DISCUSSION TITLE
Blazor Components for Material Design Web Components

### DISCUSSION THREAD

**Discussion by datvm** [material-components/material-web#5023] [Show and tell]
I am very happy they released v1.0.0. I am working on [BlazorMaterialWeb](https://github.com/datvm/BlazorMaterialWeb) that wraps these components and today I finally caught up to the changes. You can check it out at the repo above or the demo page at: https://blazorwebmaterial.lukevo.com/ . Everything is built and hosted by Github Actions.

The above project also use a side project at https://github.com/datvm/material-web that builds and makes a single file ESM bundle from this project so it can be used without any build tool. It's also built by Github Actions.

**Comment by asyncliz** [comment]
Awesome work!
